Add `Skinning Contest: Tides of Winter` Newspost (#6472)
ppy/osu-wiki: news/2021-12-02-skinning-contest-tides-of-winter-accouncement.md

---
layout: post
title: "Skinning Contest: Tides of Winter"

> date: 2021-12-02 12:00:00 +0000

Reviewer: collides with https://github.com/ppy/osu-wiki/pull/6467

Author: @Ephemeralis

Reviewer 2: fwiw, as long as it doesn't collide with FA newsposts, it should be fine.

Reviewer: exhibit A:
![image](https://user-images.githubusercontent.com/36758269/144095070-84b4c965-6687-454c-9417-577d0c14153f.png)

exhibit B:
![image](https://user-images.githubusercontent.com/36758269/144095185-dabf2725-8482-436b-a17e-9b6c472ab543.png)

maybe we're fine, idk

Author: it maybe is fine, but its still bad to bump someone from the topspot of the homepage when there are other open spots. The newspost goes too quickly thanks to FA still not being split off (https://github.com/ppy/osu-web/issues/7685), best to prevent collides wherever possible. Tho that being said, pushing it by a day would mean closing it on the 31st, which does suck, so guess people would only get 27days instead of 28 since we would still need to close on the 30th

Reviewer 2: I'd say those are either an exception or an oversight, but let's wait for eph to chime in and give us a clear answer.

Reviewer 3: clipping on non FA stuff is fine, though ideally avoided

we can probably redate this to roughly 12 hours from now, don't believe anything else is fighting for the slot

---

> The Skinning Contest returns as Winter cracks the door open. Grab a cup of hot chocolate and tune in for all the details!

Reviewer: ```suggestion
The Skinning Contest returns as winter cracks the door open. Grab a cup of hot chocolate and tune in for all the details!
```

![](/wiki/shared/news/2021-12-02-skinning-contest-tides-of-winter-accouncement/announcement.jpg)

> Months after the launch of our first iteration, we're once again here to call forth all the skinners for the second time to show off their talents in the Skinning Contest, hosted by one of the largest skinning communities, [skinship](https://skinship.xyz/). For those unfamiliar with what this is all about, check out the wiki page for the previous iteration [here](/wiki/Contests/Skinning_Contest/1).

Reviewer: style

```suggestion
Months after the launch of our first iteration, we're once again here to call forth all the skinners for the second time to show off their talents in the Skinning Contest, hosted by one of the largest skinning communities, [skinship](https://skinship.xyz/). For those unfamiliar with what this is all about, check out the [wiki page](/wiki/Contests/Skinning_Contest/1) for the previous iteration.
```

> Since the Winter is here and the snowy days are nearing, we have decided to adopt a theme that will reflect the season properly: **Tides of Winter**! As you might have guessed, in this iteration of the contest, we expect submissions all themed around anything Winter-related. Whether it fills people joy with the everlasting Christmas spirit, or pulls out a different take on Winter sports, anything is acceptable as long as it fits the theme.

Reviewer: ```suggestion
Since winter is here and the snowy days are nearing, we have decided to adopt a theme that will reflect the season properly: **Tides of Winter**! As you might have guessed, in this iteration of the contest, we expect submissions all themed around anything winter-related. Whether it fills people with joy and everlasting Christmas spirit, or pulls out a different take on winter sports, anything is acceptable as long as it fits the theme.
```

## Changes

As per the feedback we've received from you, we've decided to make a couple of adjustments and improvements to this edition of the contest:

- Each contestant can only participate with their own work, meaning that **collaborations/teams are not allowed**. Of course, you are permitted to ask others for feedback, but that should not go beyond exchanging ideas.

> - The file size of your skin as a whole must be **under 100MB**. As much as we like high-fidelity visual and aural content, we have to look out for those who have a poor system or internet connection.

Reviewer: ```suggestion
- The file size of your skin as a whole must be **under 100 MB**. As much as we like high-fidelity visual and aural content, we have to look out for those who have a poor system or internet connection.
```

All of the changes above can be viewed in detail in the [Rules](#rules) section below. Make sure to have a read!

## Participation

To enter the competition, you must create a brand-new skin from scratch, ready to compete against skin creators from all over the world!

In order for your submission to be valid, and also for everyone else to fall in love with your creation, you need to create a forum thread accompanying the skin. However you advertise your skin, though, is completely up to you. Said forum thread must be located under the [Skinning WIP subforum](https://osu.ppy.sh/community/forums/119).

Once you have prepared your masterpiece, all you have to do is to leave a post under the [submission thread](https://osu.ppy.sh/community/forums/topics/1472846)!

Since all entries will have their respective forum threads publicly available, contestants and their submissions will not be anonymized. In fact, we encourage you to show off your work as much as you wish!

## Voting

After the submission phase is over, the entries will be made available for community voting.

> The results of the community voting will determine the winner of the contest. Entry with the most votes wins!

Reviewer: ```suggestion
The results of the community voting will determine the winner of the contest. The entry with the most votes wins!
```

## Schedule

- Submission phase: **4** weeks starting from the release of this newspost
- Voting phase: **2** weeks starting from a separate newspost

## Rules

- Submissions must not contain inappropriate, malicious, or epileptic content, and should adhere to the [osu! community rules](/wiki/Rules).
- All assets need to be created by yourself, or used with permission and proper credit to the authors. This includes, but is not limited to:
  - fonts
  - icons
  - textures
  - artworks (e.g. stock images or anime artwork)
  - sounds
- The majority of the in-game menu interface and at least two game modes have to be skinned. In other words, in addition to the gameplay elements, the following segments shouldn't be left default:
  - Ranking Panel
  - Song Selection
  - Mode Selection
  - Mod Icons
  - Pause and Fail Menu

>   - Main Menu is optional, but highly encouraged

Reviewer: ```suggestion
  - Main Menu is optional, but highly encouraged.
```

Author: This isn't a full sentence, just a bullet point, so no . should be needed

Reviewer: It *is* a full sentence, but I guess it can stay as is.

- Submissions must be new creations, only skins posted within the submission period will be accepted.
- Each submission must be accompanied by a forum thread in order to participate in the contest.
- Submissions must be made in time. Latecomers will not be accepted.

> - Submissions must tbe created individually. Teams/Collaborations are not allowed.

Reviewer: ```suggestion
- Submissions must be created individually. Teams/collaborations are not allowed.
```

> - Submissions must be in line with the given theme: Tides of Winter

Reviewer: ```suggestion
- Submissions must be in line with the given theme: "Tides of Winter".
```

> - The file size for your .osk submission must be below 100MB.

Reviewer: ```suggestion
- The file size for your .osk submission must be below 100 MB.
```

> - Please submit a 16:9 image representing your skin, this will be used as a cover on the contest page.

Reviewer: ```suggestion
- Please submit a 16:9 image representing your skin, since this will be used as a cover on the contest page.
```

**Failure to comply with the contest rules may lead to disqualification, or, in severe cases, exclusion from future contests.**

## Prizes

> Now that all the rules are our of the way, let's unveil the prizes delivered to the best of the best!

Reviewer: ```suggestion
Now that all the rules are out of the way, let's unveil the prizes that will be delivered to the best of the best!
```
